retriever/lib/engine_tools.py

```diff
@@ -131,7 +131,6 @@ def json2csv(input_file, output_file=None, header_values=None, encoding=ENCODING
     subprocess.call(['rm', '-r', input_file])
     return output_file
 
-
```

Comment:
_Deleted lines_
I do  not recommend changing this here. 
If there are some lines that need to be removed, we can create a new PR separate from this since the changes are not closely coupled to this Kaggle PR

Reply:
I had a vscode extension running that deleted excess spaces. I have removed the changes